pandas/core/arrays/datetimelike.py

```diff
@@ -728,7 +728,7 @@ def _maybe_mask_results(self, result, fill_value=iNaT, convert=None):
         ----------
         result : a ndarray
         fill_value : object, default iNaT
-        convert : string/dtype or None
+        convert : str/dtype or None
```

Comment:
`str, dtype or None` would be better. Besides consistency, in the future I'd like to be able to parse the types.